tests/unit/test_adapter.py

```diff
@@ -238,6 +238,7 @@ def test_ldap_authentication(self):
                 "method": "ldap",
                 "schema": "dbt_test_schema",
                 "user": "trino_user",
+                "impersonation_user": "impersonated_user",
```

Comment:
Is there a place I can assert the user we pass into dbapi ?

Reply:
Just please add `assertEqual` for `impersonation_user` below.